client/src/components/BookATripImage/BookATripImage.js

```diff
@@ -10,10 +10,10 @@ const BookATripImage = (props) => {
       <div className="BookATripImage__image-group">
         {cards.map(card => {
           return (
-            <div className="BookATripImage__image-container">
-              <img src={require(`../../assets/images/${card.image}.png`)} className="BookATripImage__image"></img>
-              <div><Link to={card.url} className="BookATripImage__link">{card.description}</Link></div>
-            </div>
+              <div className="BookATripImage__image-container">
+                <img src={require(`../../assets/images/${card.image}.png`)} className="BookATripImage__image"></img>
```

Comment:
This is very clever! However, these images may come from the backend at some point. So it'd be better to accept the image as a prop